app/parsers/bulkrax/xml_parser.rb

```diff
@@ -12,11 +12,23 @@ def collection_entry_class; end
     # @todo not yet supported
     def create_collections; end
 
+    def create_file_sets; end
```

Comment:
the empty methods in this file are to override (or set back) the now defaults in application_parser.rb.

Reply:
Instead of "unsetting" defaults, should the bagit parser just inherit from the csv parser? (see [related comment](https://github.com/samvera-labs/bulkrax/pull/571#discussion_r902779855))